decoder/gym.go

```diff
@@ -374,11 +376,77 @@ func createGymFortWebhooks(oldGym *Gym, gym *Gym) {
 	}
 }
 
+func makeRaidWebhook(gym *Gym) (payload map[string]interface{}) {
+	payload = map[string]interface{}{
+		"gym_id": gym.Id,
+		"gym_name": func() string {
+			if !gym.Name.Valid {
+				return "Unknown"
+			} else {
+				return gym.Name.String
+			}
+		}(),
+		"gym_url":                gym.Url.ValueOrZero(),
+		"latitude":               gym.Lat,
+		"longitude":              gym.Lon,
+		"team_id":                gym.TeamId.ValueOrZero(),
+		"spawn":                  gym.RaidSpawnTimestamp.ValueOrZero(),
+		"start":                  gym.RaidBattleTimestamp.ValueOrZero(),
+		"end":                    gym.RaidEndTimestamp.ValueOrZero(),
+		"level":                  gym.RaidLevel.ValueOrZero(),
+		"pokemon_id":             gym.RaidPokemonId.ValueOrZero(),
+		"cp":                     gym.RaidPokemonCp.ValueOrZero(),
+		"gender":                 gym.RaidPokemonGender.ValueOrZero(),
+		"form":                   gym.RaidPokemonForm.ValueOrZero(),
+		"alignment":              gym.RaidPokemonAlignment.ValueOrZero(),
+		"costume":                gym.RaidPokemonCostume.ValueOrZero(),
+		"evolution":              gym.RaidPokemonEvolution.ValueOrZero(),
+		"move_1":                 gym.RaidPokemonMove1.ValueOrZero(),
+		"move_2":                 gym.RaidPokemonMove2.ValueOrZero(),
+		"ex_raid_eligible":       gym.ExRaidEligible.ValueOrZero(),
+		"is_exclusive":           gym.RaidIsExclusive.ValueOrZero(),
+		"sponsor_id":             gym.SponsorId.ValueOrZero(),
+		"partner_id":             gym.PartnerId.ValueOrZero(),
+		"power_up_points":        gym.PowerUpPoints.ValueOrZero(),
+		"power_up_level":         gym.PowerUpLevel.ValueOrZero(),
+		"power_up_end_timestamp": gym.PowerUpEndTimestamp.ValueOrZero(),
+		"ar_scan_eligible":       gym.ArScanEligible.ValueOrZero(),
+	}
+	if gym.LobbyPlayerCount > 0 {
```

Comment:
Here 0 essentially means null. Do you want to always populate 0 instead?

Reply:
If it is actually 0, then that makes sense. If you don't know, then it should be null.
I'm not really sure what this will look like with real data, so it might be better to hold this one over until we have a data source.

Reply:
I wonder if it should be null until you have received the first lobby for a gym; then when the lobby times out go to 0? or go back to null?

Reply:
Personally I think it's fine to keep it 0 always to keep it simple (instead of having to deal with 0 vs null), due to the extremely ephemeral nature of raid lobbies (only lasting at most 2 minutes).

Reply:
Not sure what you are talking about but it is impossible to have an active lobby that is also empty.

Reply:
This hook will be sent again by any other raid related changes, and the discussion is about these fields always being present and the values they should have?

Reply:
Yes and I think in that case having 0 is also reasonable? 0 = empty = inactive = no lobby, and it might as well represent unknown/expired lobbies?

Reply:
Even if there was a lobby, it might never get scanned when it was active. Therefore to me, it seems pointless to distinguish whether the lobby was last seen empty/expired vs the lobby was never scanned since raid started.

Reply:
I am thinking about downlevel users of the raid webhook. If we are sending old lobby details that's a bit pointless, eh? Any users of the data in eg a poracle notification would therefore see a lobby that wasn't actually correct.

Reply:
Poracle can check if the lobby expired when it actually gets to it in the queue.

Reply:
That's not how poracle works.  The webhook information is available to users to put in their messages without processing; with some additions.  We can hold this open until we get a data source and some implementors, but the current implementation is inconsistent with every other webhook

Reply:
If you don't like it you have the freedom to change it.